agent/pkg/models/models.go

```diff
@@ -16,6 +16,14 @@ func GetEntry(r *tapApi.MizuEntry, v tapApi.DataUnmarshaler) error {
 	return v.UnmarshalData(r)
 }
 
+type EntriesRequest struct {
```

Comment:
I'm thinking if we need to specify the "max" in the validation.. 
it will probably give better errorMessage but if the validate package is working well with specify only min, it might be enought. (and in case of bigger number we will probably got exception when trying to assign this number to the struct

Reply:
Done with https://github.com/up9inc/mizu/commit/9e6919a01c1e9e7cdb253636a26560d275b49ea6